IframeEmbedding/Controllers/HomeController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using IframeEmbedding.Models;
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using System.Web.Mvc;
+
+namespace IframeEmbedding.Controllers
+{
+    public class HomeController : Controller
+    {
+        public ActionResult Index()
+        {
+            var nonce = Guid.NewGuid().ToString(); // random string
```

Comment:
Variable declaration is not good. Get the detail and knowledge from Nithya as she worked on Iframe SSO using token Embedding sample.

Reply:
updated all variables